example/javalib/linting/3-palantirjavaformat/build.mill

```diff
@@ -0,0 +1,23 @@
+package build
+import mill._, javalib._
+import $ivy.`com.lihaoyi::mill-contrib-palantirjavaformat:`
+
+import mill.contrib.palantirjavaformat._
+
+object `package` extends RootModule with PalantirJavaFormatModule {
+}
+
+/** Usage
+
+> ./mill javafmt --check                # check Java source files for formatting errors
+...: checking format in java sources ...
+.../src/A.java
+...: palantirjavaformat aborted due to format error(s) (or invalid plugin settings/palantirjavaformat options)
```

Comment:
If this returns an exit code != 0, you need to prefix the first line of output with `error: ` to tell the example testing framework to expect the failure